OracleWebLogic/samples/wls-k8s-domain/Dockerfile

```diff
@@ -0,0 +1,36 @@
+
+# Pull base image
+# ---------------
+FROM weblogic-12.2.1.3-developer:latest
+
+MAINTAINER Lily He <lily.he@oracle.com>
+
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+ENV MW_HOME="$ORACLE_HOME" \
+    PATH="$ORACLE_HOME/wlserver/server/bin:$ORACLE_HOME/wlserver/../oracle_common/modules/org.apache.ant_1.9.2/bin:$JAVA_HOME/jre/bin:$JAVA_HOME/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:$ORACLE_HOME/oracle_common/common/bin:$ORACLE_HOME/wlserver/common/bin:$ORACLE_HOME/wlserver/samples/server/" \
```

Comment:
This `PATH` environment variable is way too long for a downstream image. Remember you inherit from the previous image, so it should be simplified to:

```
PATH="$PATH:$ORACLE_HOME/wlserver/samples/server/:$ORACLE_HOME/wlserver/server/bin:$ORACLE_HOME/wlserver/../oracle_common/modules/org.apache.ant_1.9.2/bin
```

The rest will all be inherited in the `$PATH` variable that is provided by the `weblogic:12.2.1.3-dev` image.